qemux86-64.robot

```diff
@@ -39,17 +39,23 @@ Checking if device comes online in 60s (Trying every 10s)
   Wait Until Keyword Succeeds    6x    10s    Device "${device_uuid}" is online
 Verify if resin-info@tty1.service is active
   Check if service "resin-info@tty1.service" is running using socket "unix\#/tmp/console.sock"
-Check that backup files are not found in the image
-  Check that backup files are not found in the "${image}"
 Git pushing to application
-  Push "${application_repo}":"${application_commit}" to application "${application_name}"
+  Git clone "${application_repo}" "/tmp/${application_name}"
+  Git checkout "${application_commit}" "/tmp/${application_name}"
+  Git push "/tmp/${application_name}" to application "${application_name}"
 Check if device is running the pushed application (Tries for 300 s)
   Wait Until Keyword Succeeds    30x    10s    Device "${device_uuid}" log should contain "Hello"
 Check if kernel module loading works
   Check if kernel module loading works on "${device_uuid}"
+Check delta to a running supervisor
+  Check enabling supervisor delta on "${application_name}"
 Check if resin sync works
   Check if resin sync works on "${device_uuid}"
 Check if setting environment variable works
   Check if setting environment variables works on "${application_name}"
 Verify if host OS version of the image is same through resin cli
   Check if host OS version of device "${device_uuid}" is "${os_version}"
+Wait till Qemu is killed or 30 seconds
```

Comment:
Missed this - We need to call a device shutdown before waiting for the process to exit - right now the timeout will always occur and the qemu instance will be killed
`resin device shutdown` command could be incorporated in resincli.robot and used here